packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -131,6 +133,20 @@ class PaginatedDataTable extends StatefulWidget {
   /// See [DataTable.onSelectAll].
   final ValueSetter<bool> onSelectAll;
 
+  /// The horizontal padding between the edges of the table and the content
+  /// in the first and last cells of each row.
+  ///
+  /// When a checkbox is displayed, it is also the padding between the checkbox
+  /// the content in the first data column.
+  ///
+  /// This value is optional and defaults to 24.0.
+  final double tablePadding;
+
+  /// The horizontal padding between the contents of each data column.
+  ///
+  /// This value is optional and defaults to 56.0.
+  final double columnSpacing;
```

Comment:
See [comment from your other PR](https://github.com/flutter/flutter/pull/33535#discussion_r292583576) about commenting on this "default".